docs/template/public/less/_common.less

```diff
@@ -0,0 +1,147 @@
+// Common styles
+
+@import '_variables';
+
+body.kss-body {
+    background: @white;
+    overflow: visible;
+    font-size: 15px;
+    font-family: 'Open Sans', sans-serif;
+    padding: 0;
+    max-width: 100%;
+    padding-top: 100px;
+    padding-right:0!important;
+}
+
+#kss-wrapper {
+    height: auto;
+}
+
+.active.demo-container {
+    background-color: @screen-bg_gd;
+    padding: 15px;
+}
+
+#kss-header h1 {
+    font-size: 1.5em;
+    margin-top: 5/3em;
+    margin-left: 4/3em;
+}
+
+h1.kss-title {
+    color: @blue-heading;
+    margin-bottom: 30px;
+    font-weight: 500;
+    font-family: 'Dosis', sans-serif;
+}
+
+#kss-main {
+    padding-left: 15px;
+    padding-bottom: 40px;
+
+    &.col-md-12{
+        max-width: 1000px;
```

Comment:
Making this change is breaking the home page as it is nested with rest of the pages with the index template. I don't know if we should break it away from the template? 

<img width="1436" alt="screen shot 2016-06-27 at 10 28 40 am" src="https://cloud.githubusercontent.com/assets/15215400/16385368/1bd477d6-3c52-11e6-89d6-612e6bad9858.png">